addons/udes_stock_refactoring/data/stock_picking_types.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+     <!--Do we want to to change the u_move_line_key_format to {package_id.id}?  -->
```

Comment:
Perhaps rename this file to 'picking_types.xml' instead? Fits with current convention in UDES 11.

Reply:
Yep will rename this.

Reply:
This has now been renamed.